ui/src/components/Ec2ResourcesTable/index.jsx

```diff
@@ -109,22 +109,40 @@ export default class Ec2ResourcesTable extends Component {
       sortDirection
     );
     const iconSize = 16;
+    const headers = [
+      { label: 'Instance ID', id: 'instanceId', type: 'string' },
+      {
+        label: 'State',
+        id: 'state',
+        type: 'string',
+      },
+      // Blocked by https://bugzilla.mozilla.org/show_bug.cgi?id=1453649
+      // 'Instance Type',
```

Comment:
We are no longer blocked by this bug. We can remove the comment.

Reply:
Should I also remove the 'Instance Type' and 'AMI' comments? What are those for?

Reply:
Yeah, you can remove them.